### DISCUSSION TITLE
Floating Container Behavior

### DISCUSSION THREAD

**Discussion by arakis** [i3/i3#5764] [Q&A] [answered]
Sequence:

mod+Return
mod+Return
mod+a
Shift+mod+Space
--> - Split Container with two terminals
    - mod+Return adds a third terminal
    - **PROBLEM**: it seems it's not possible to create a regular terminal tiling terminal in the background (behind the floating window)

What works:
- Keep at least one tiling window:

mod+Return
mod+Return
mod+h
mod+Return
mod+a
Shift+mod+Space
--> - Now you're able to change focus between floating window and tiling window, incl. creating additional windows.
    - **That's perfect!**

Another strange situation:

mod+Return
Shift+mod+Space
mod+Return
--> - Now it was possible to create a tiling termin in the background,
    after the floating was created.
    - But now it's **impossible** to convert the floating window into a container.
    
Version: 4.22, on Arch Linux

**Comment by arakis** [comment, marked as answer]
Is this behavior intended, or should I create a bug report?

**Reply by orestisfl** [reply]
The bug report: https://github.com/i3/i3/issues/5793
